10-message-translations.Rmd

```diff
@@ -352,4 +352,5 @@ Instead of translating one string at a time, it is possible to bulk translate a
 
 ## Helpful references
 
--   Statistical terms glossary
```

Comment:
This line was already here before this PR, though looks like it should link to somewhere, do you know what the link should be @hturner? Happy to add it in with these changes

Reply:
I think this should be https://www.isi-web.org/glossary, which provides translations in over 25 languages.